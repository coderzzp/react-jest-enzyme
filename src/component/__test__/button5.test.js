// mocktimer
import React from 'react'
import { shallow } from 'enzyme';

jest.useFakeTimers()

describe('Button 组件测试', () => {
  it('3秒后渲染正常', () => {
    const Button = require('../button').default
    const componentWrapper = shallow(<Button />);
    jest.runAllTimers()
    expect(componentWrapper.html()).toMatchSnapshot();
  });
  
});