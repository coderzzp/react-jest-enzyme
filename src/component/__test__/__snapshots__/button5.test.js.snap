// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 组件测试 3秒后渲染正常 1`] = `"<div><div class=\\"button\\">buttonAfter3seconds</div></div>"`;
